ekncontent/overrides/EosKnowledgeContent.js

```diff
@@ -89,7 +89,9 @@ function _init() {
     });
     define_property(Eknc.ArticleObjectModel, 'table-of-contents', {
         get: function () {
-            let toc = this.get_table_of_contents().deep_unpack();
+            let toc = this.get_table_of_contents();
+            if (toc)
+                toc = toc.deep_unpack();
```

Comment:
Just move this line under the if statement below, and you don't need this second if right?